src/core/evaluator.js

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2012 Mozilla Foundation
+  /* Copyright 2012 Mozilla Foundation
```

Comment:
Thanks, it certainly shouldn't be there, good catch! I've got no idea how that happened :-)